src/lib_c/x86_64-freebsd/c/signal.cr

```diff
@@ -8,31 +8,33 @@ lib LibC
   SIGILL    = 4
   SIGTRAP   = 5
   SIGIOT    = LibC::SIGABRT
-  SIGABRT   =  6
-  SIGFPE    =  8
-  SIGKILL   =  9
-  SIGBUS    = 10
-  SIGSEGV   = 11
-  SIGSYS    = 12
-  SIGPIPE   = 13
-  SIGALRM   = 14
-  SIGTERM   = 15
-  SIGURG    = 16
-  SIGSTOP   = 17
-  SIGTSTP   = 18
-  SIGCONT   = 19
-  SIGCHLD   = 20
-  SIGTTIN   = 21
-  SIGTTOU   = 22
-  SIGIO     = 23
-  SIGXCPU   = 24
-  SIGXFSZ   = 25
-  SIGVTALRM = 26
-  SIGUSR1   = 30
-  SIGUSR2   = 31
-  SIGEMT    =  7
-  SIGINFO   = 29
-  SIGWINCH  = 28
+  SIGABRT   =   6
+  SIGFPE    =   8
+  SIGKILL   =   9
+  SIGBUS    =  10
+  SIGSEGV   =  11
+  SIGSYS    =  12
+  SIGPIPE   =  13
+  SIGALRM   =  14
+  SIGTERM   =  15
+  SIGURG    =  16
+  SIGSTOP   =  17
+  SIGTSTP   =  18
+  SIGCONT   =  19
+  SIGCHLD   =  20
+  SIGTTIN   =  21
+  SIGTTOU   =  22
+  SIGIO     =  23
+  SIGXCPU   =  24
+  SIGXFSZ   =  25
+  SIGVTALRM =  26
+  SIGUSR1   =  30
+  SIGUSR2   =  31
+  SIGEMT    =   7
+  SIGINFO   =  29
+  SIGWINCH  =  28
+  SIGRTMIN  =  65
+  SIGRTMAX  = 126
```

Comment:
Let's note that SIGRTMIN and SIGRTMAX also exist on:

- Linux: but are function calls `__libc_current_sigrtmin()` and `__libc_current_sigrtmin()`;
- Solaris: but are calling sysconf: `_sysconf(_SC_SIGRT_MIN)` and `_sysconf(_SC_SIGRT_MAX)`;
- NetBSD: but kernel only (not exposed to userland yet).